app/src/main/res/layout/activity_account.xml

```diff
@@ -157,6 +157,17 @@
                     android:layout="@layout/view_account_moved"
                     app:layout_constraintTop_toBottomOf="@id/accountFieldList" />
 
+                <TextView
+                    android:id="@+id/accountRemoveView"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginBottom="8dp"
+                    android:text="@string/label_remote_account"
+                    android:visibility="gone"
+                    app:layout_constraintTop_toBottomOf="@id/accountMovedView"
```

Comment:
can you please give it a `android:lineSpacingMultiplier="1.1"` like on `accountNoteTextView`

Reply:
Thanks for the review!
No problem, if it looks better to you, I was just afraid that it will be too big